pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/endpoints/objects_api/memberships/SetMembershipsBuilder.java

```diff
@@ -0,0 +1,17 @@
+package com.pubnub.api.java.endpoints.objects_api.memberships;
+
+import com.pubnub.api.java.endpoints.Endpoint;
+import com.pubnub.api.java.endpoints.objects_api.utils.PNSortKey;
+import com.pubnub.api.java.models.consumer.objects_api.membership.MembershipInclude;
+import com.pubnub.api.java.models.consumer.objects_api.membership.PNSetMembershipResult;
+
+import java.util.Collection;
+
+public interface SetMembershipsBuilder extends Endpoint<PNSetMembershipResult> {
```

Comment:
do we need this new interface at all?
We could just make changes to `interface SetMemberships`

Reply:
We don't need new interface but I thought it will be more convenient for user to deal only with valid options. There are 4 parameters in interface `SetMemberships` that should be deprecated.

What do you think?